File: Java/Lista 3/L3_Ex1.4/src/Usuario.java

public class Usuario {
    private String login;

    public Usuario(String login) {
        this.login = login;
    }

    public String getLogin() {
        return login;
    }
    
    public void boasvindas(){
        System.out.println("Bem-vindo " + this.login );
    }        
     
}

File: Swift/L4/Lista_04_EX01/Usuario.swift
public class Usuario {
    private login: String

    init(login: String) {
        self.login = login
    }

    func getLogin() -> String {
        return login;
    }
    
    func menu() -> int {
        var opcao = 0
        println("Menu")
        return opcao
    }
    
    func boasvindas(){
        println("Bem-vindo \(self.login)")
    }
}